packages/next-app/src/app/layout.tsx

```diff
@@ -13,7 +13,7 @@ export const metadata = {
 export default function Layout({ children }: { children: React.ReactNode }) {
   return (
     <html lang="en" className="h-full">
-      <body className="h-full flex flex-col bg-blue2">
+      <body className="flex h-full min-w-[930px] flex-col bg-blue2">
```

Comment:
on phones this would look super weird right?

Reply:
From what I remembered on the beginning of Internal Manager is that our apps are not meant for phones rn. Are we going to tackle that?